### ISSUE TITLE
Follow meta refresh tags (when BrowserKit does)

### ISSUE THREAD

**Issue by jhedstrom** [Behat/MinkExtension#326]
There is an open PR in BrowserKit to follow meta refresh tags (https://github.com/symfony/symfony/pull/27118). Once this is merged, it would be nice to have a configurable parameter for `Goutte` to enable that option.

This relates to https://github.com/jhedstrom/drupalextension/pull/325#issuecomment-349724176
